deltachat-ios/Controller/AccountSetup/Instand Onboarding/InstantOnboardingViewController.swift

```diff
@@ -238,9 +238,11 @@ class InstantOnboardingViewController: UIViewController {
     }
 
     private func handleCreateSuccess() {
-        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
-        appDelegate.registerForNotifications()
-        appDelegate.reloadDcContext()
+        DispatchQueue.main.async {
+            guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+            appDelegate.registerForNotifications()
+            appDelegate.reloadDcContext()
+        }
```

Comment:
took me a moment to verify that handleCreateSuccess() is called only from main, good catch!